gooddata-java-model/src/main/java/com/gooddata/sdk/model/projecttemplate/Template.java

```diff
@@ -19,7 +19,12 @@
 /**
  * Represents one project template.
  * Deserialization only.
+ *
+ * @deprecated The project templates are deprecated and stopped working on May 15, 2019.
+ * See https://support.gooddata.com/hc/en-us/articles/360016126334-April-4-2019
+ * Deprecated since version 3.0.0. Will be removed in one of future versions.
```

Comment:
Well, we already released the `3.0.0` so we have 2 options:
* It was missing in the `3.0.0` so it's a bugfix
* It's a new feature, so we should say that it's deprecated from `3.1.0`

Reply:
Well, even if I make bugfix out of it, I should change it to 3.0.1 ;)